inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/tasks/SelectionTask.java

```diff
@@ -173,7 +173,8 @@ public void run()
             recommendationService.setActiveRecommenders(user, layer, activeRecommenders);
         }
 
-        schedulingService.enqueue(new TrainingTask(user, getProject()));
+        schedulingService.enqueue(new TrainingTask(user, getProject()),
+                "Started by SelectionTask after activating recommenders");
```

Comment:
"Started by" is redundant with the "Triggered by" prefix that is always added when logging.

Reply:
Done